app/src/main/java/org/sopt/hyundaicarddive/presentation/ui/space/component/SpaceReviewPager.kt

```diff
@@ -0,0 +1,107 @@
+package org.sopt.hyundaicarddive.presentation.ui.space.component
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.PageSize
+import androidx.compose.foundation.pager.PagerState
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import org.sopt.hyundaicarddive.R
+import org.sopt.hyundaicarddive.core.util.noRippleClickable
+import org.sopt.hyundaicarddive.presentation.model.SpaceReviewModel
+import org.sopt.hyundaicarddive.ui.theme.HYUNDAICARDDIVETheme
+import org.sopt.hyundaicarddive.ui.theme.HYUNDAICARDDIVETheme.colors
+import org.sopt.hyundaicarddive.ui.theme.HYUNDAICARDDIVETheme.typography
+
+@Composable
+fun SpaceReviewPager(
+    spaceReviewList: List<SpaceReviewModel>,
+    pagerState: PagerState,
+    onClickLike: (Int) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    HorizontalPager(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(bottom = 24.dp),
+        state = pagerState,
+        pageSize = PageSize.Fill,
+        contentPadding = PaddingValues(horizontal = 20.dp)
+    ) { page ->
+
+        if (page == spaceReviewList.size) {
+            Spacer(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .aspectRatio(292f / 200f)
+            )
+        } else {
+            Column(
+                modifier = Modifier
+                    .padding(end = 20.dp)
+            ) {
+                AsyncImage(
+                    model = spaceReviewList[page].imgUrl,
+                    contentDescription = null,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .aspectRatio(292f / 200f)
+                )
+                Row(
+                    modifier = Modifier
+                        .padding(top = 12.dp),
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Text(
+                        text = spaceReviewList[page].reviewer + " 님",
```

Comment:
하드코딩 머에요 ㅡㅡ

이거 string 파일에 예를 들어
```
    <string name="space_review_list_reviewer">%s 님</string>
```
이런 식으로 저장해두고 
```suggestion
                        text= stringResource(R.string.space_review_list_reviewer, spaceReviewList[page].reviewer)
```
이런 식으로 하드코딩을 방지할 수 있답니다 ~

Reply:
딱 들켰다.. 바로 수정할게요 ..!